acceptance/openstack/blockstorage/v3/blockstorage.go

```diff
@@ -145,6 +211,20 @@ func DeleteVolume(t *testing.T, client *gophercloud.ServiceClient, volume *volum
 		t.Fatalf("Unable to delete volume %s: %v", volume.ID, err)
 	}
 
+	// VolumeTypes can't be deleted until their volumes have been,
```

Comment:
This change affects all tests that use DeleteVolume, while its only sort of needed when extra resources depend on the volume such as a volumeType. Let me know if a seperate DeleteVolumeAndWait func is better

Reply:
This is a good addition. Thank you!